public/app/features/plugins/sql/components/configuration/TLSSecretsConfig.tsx

```diff
@@ -23,60 +24,94 @@ export const TLSSecretsConfig = <T extends DataSourceJsonData, S extends {} = {}
   return (
     <>
       {showKeyPair ? (
-        <InlineField
-          tooltip={
-            <span>To authenticate with an TLS/SSL client certificate, provide the client certificate here.</span>
+        <Field
+          label={
+            <Label>
+              <Stack gap={0.5}>
+                <span>TLS/SSL Client Certificate</span>
+                <Tooltip
+                  content={
+                    <span>
+                      To authenticate with an TLS/SSL client certificate, provide the client certificate here.
+                    </span>
+                  }
+                >
+                  <Icon name="info-circle" size="sm" />
+                </Tooltip>
+              </Stack>
+            </Label>
           }
-          labelWidth={labelWidth}
-          label="TLS/SSL Client Certificate"
         >
-          <SecretTextArea
-            placeholder="Begins with -----BEGIN CERTIFICATE-----"
+          <SecretInput
```

Comment:
Do we still want this to be a larger text field?

Reply:
thats a fair point, i feel like there was a reason i changed it but i'm not sure what that was 😆. ill change it back.